### ISSUE TITLE
Usage

### ISSUE THREAD

**Issue by jamesdixon** [estate/bookshelf-history#4]
Hi, guys.

I just stumbled across this and was wondering how you're currently using it?

I've run into a situation where I need to be able to be able to essentially archive models for historical purposes and wondering if this addon might fit the bill. The situation is this:

The platform I'm building deals primarily with appointments. Each appointment has a service associated with it. Business owners may want to change details about their services from time-to-time, but I need to be able to retain the existing service before modifications are made so that a change to a service wouldn't reflect in a past appointment. 

Any thoughts on whether or not your addon sounds like a good fit?

Thanks!
James

**Comment by alanhoff** [comment]
@jamesdixon sounds like a good application for bookshelf-history.

**Comment by bsiddiqui** [comment]
Hey @jamesdixon we aren't currently using `bookshelf-history` because we stopped needing to store historical records so [bookshelf-paranoia](https://github.com/estate/bookshelf-paranoia) was enough.

It sounds like this plugin might work well for you. Whenever a change is made to a model that is being tracked with a history table, the model will update and a row will be added to the history table so you have a historical reference. We made it pretty simple for you to revert back to a previous point in history.
